app/apollo/subscription/index.js

```diff
@@ -83,6 +84,8 @@ class PubSubImpl {
       logger.info(
         `Apollo streaming is enabled on redis endpoint ${url.hostname}:${url.port}`,
       );
+      this.livenessCheckPublisher();
+      this.livenessCheckSubscriber(this.pubsubQueue);
```

Comment:
we do not need parameter, you could get variable from this.variablename in the function.